src/Dotnet.Status.Web/DotNet.Status.Web.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <UserSecretsId>65a272ca-730a-4afd-b68b-d256719c2db2</UserSecretsId>
+    <SignAssembly>false</SignAssembly>
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GitHubJwt" Version="0.0.2" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0" />
+    <PackageReference Include="Azure.Storage.Files" Version="12.0.0-preview.3" />
```

Comment:
>"12.0.0-preview.3" [](start = 60, length = 18)

You updated Blobs and Queues, why not Files?

Reply:
I don't use files. I'll probably just remove it.

---
In reply to: [344318121](https://github.com/dotnet/arcade-services/pull/756#discussion_r344318121) [](ancestors = 344318121)